pkg/defaults/defaults.go

```diff
@@ -246,6 +246,14 @@ const (
 	// the endpoint could send/receive traffic to/from.
 	PolicyMapEntries = 16384 // Cilium 1.5 and earlier value
 
+	// PolicyMapMinSize is the minimum value to which the number of entries
```

Comment:
This will conflict with the refactoring I did in #10957 (for #10780) where I moved these consts to the top of `pkg/option/config` (like the other map min/max constants). Same for `FragmentMapMaxSize`. I think we can keep them in `pkg/option` as they are not used outside of the package.

I think it makes sense to keep the default map sizes in `pkg/defaults` though.

Reply:
With your change they would be in a list of similar limits for the other maps, which looks better than leaving them in the middle of the `config` package as it is now, so OK.

I'll probably hold until your PR is merged in that case, and rebase afterwards.

Thanks!